src/containers/Login.vue

```diff
@@ -345,22 +305,19 @@ export default Vue.extend({
   padding: 10px 30px;
   border-radius: 6px;
 }
-.plan {
-  padding: 59px 0px 59px 104px;
-}
 .plan-head {
   padding-bottom: 30px;
   font-size: 60px;
   line-height: 60px;
-  color: #4f4f4f;
+  color: white;
```

Comment:
Can you make sure to use the color variables we have, so $white here and other places throughout the file? This would make it easy for us to update the color scheme of the site if we needed to, as we could just change the value of the variable.